### ISSUE TITLE
[OfficeOpenXml.ConditionalFormatting.ExcelConditionalFormattingCollection]

### ISSUE THREAD

**Issue by dfinke** [dfinke/ImportExcel#409]
Version 5.1.0 published on the gallery has this issue, the previous one is good.

[OfficeOpenXml.ConditionalFormatting.ExcelConditionalFormattingCollection] does not contain a method named 'Add'.

![image](https://user-images.githubusercontent.com/67258/43551618-9fdc1746-95b5-11e8-946a-2d1757f2e069.png)

**Comment by jhoneill** [comment]
https://jamesone111.wordpress.com/2018/07/30/on-powershell-parameters/ 
Strikes again. 

I refactored things to call Add-ConditionalFomatting. which is the change which causes it to go   

It gets knocked into a cocked hat by enums which have an item with a zero value  - this bug has been in Add-conditionalFormatting from the outset :-( 


So I've done the same it as I did to set format, and I've added your example to the tests  :-)  Just running a test pass and then I'll commit and create a PR.
